lib/widgets/component/directions_list.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:concordia_go/blocs/bloc.dart';
+import 'package:concordia_go/utilities/application_constants.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class DirectionsList extends StatefulWidget {
+  @override
+  State<DirectionsList> createState() => DirectionsListState();
+}
+
+class DirectionsListState extends State<DirectionsList> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: Column(
+        children: <Widget>[
+          Expanded(
+            flex: 1,
+            child: Container(
+              decoration: BoxDecoration(
+                color: concordiaRed,
+                borderRadius: BorderRadius.only(
+                  topLeft: Radius.circular(18.0),
+                  topRight: Radius.circular(18.0),
+                ),
+              ),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  Expanded(
+                    child: Align(
+                      alignment: Alignment.center,
+                      child: Text(
+                        'Directions',
+                        style: TextStyle(
+                          fontSize: 24,
+                          color: Colors.white,
+                        ),
+                      ),
+                    ),
+                  ),
+                ],
+              ),
+            ),
+          ),
+          Expanded(
+            flex: 8,
+            child: Container(
+              color: Colors.white,
+              child: BlocBuilder<DirectionsBloc, DirectionsState>(
+                builder: (context, state) {
+                  if (state is InstructionUpdate) {
+                    return ListView.builder(
+                      itemCount: state.directionsList.length,
+                      itemBuilder: (context, index) {
+                        return ListTile(
+                          title: Text(state.directionsList[index].instruction),
+                        );
+                      },
+                    );
+                  } else {
+                    return Container(
+                      height: 350,
```

Comment:
Should I be worried about this hardcoded 350?

Reply:
Yes

Reply:
Please comment on #33 that we need to do multiple device smoke tests for this ticket in Sprint 3. We will do it at the beginning of the sprint to find any bugs if any. Please comment how to get to this UI too aswell :)